style.css

```diff
@@ -482,3 +490,402 @@ h2 {
+    height: 550px;
+    background-image: url("./Images/illustration-header2.png");
+    background-size: 800px 968px;
+    background-position: 100% 30%;
+  }
+
+  .brands {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    width: 20px;
+    height: 98.96px;
+    margin-top: 150px;
+  }
+
+  .icon {
+    display: block;
+    padding: 10px;
+  }
+
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 192px 0;
+    position: absolute;
+    width: 566px;
+    height: 500px;
+    left: 143px;
+    top: 72px;
+  }
+
+  .headi {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 72px;
+    font-weight: 400;
+    line-height: 82px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-left: 235px;
+    width: 566px;
+  }
+
+  .title {
+    display: block;
+    padding: 0;
+    margin-left: 220px;
+    width: 556px;
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: auto;
+  }
+
+  p {
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 32px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: 0;
+  }
+
+  .About-myself {
+    width: 100%;
+    height: 600px;
+  }
+
+  .Recent-work {
+    height: 52px;
+    width: 100%;
+    margin-top: -60px;
+    display: flex;
+    flex-direction: row;
+    margin-left: 153px;
+  }
+
+  .line {
+    width: 794px;
+    height: 0;
+    margin-top: 150px;
+  }
+
+  .middle {
+    display: flex;
+    flex-direction: row;
+    margin-left: 185px;
+    margin-top: 70px;
+    gap: 24px;
+  }
+
+  .about-img {
+    background-image: url(./Images/Img-Placeholder4.png);
+    background-repeat: no-repeat;
+    height: 448px;
+    width: 684px;
+  }
+
+  .rightblock {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 14px;
+    height: 384px;
+    margin-top: 50px;
+  }
+
+  .Multi-Post-ul {
+    height: 40px;
+    width: 297px;
+  }
+
+  .work-container {
+    height: 448px;
+    width: 100%;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    justify-items: center;
+    align-items: center;
+  }
+
+  .art-container {
+    height: 435px;
+    width: 369px;
+    margin-left: 142px;
+    background-image: url("./Images/Img-Placeholder10.png");
+  }
+
+  .art-container:hover {
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
+  .art-container1 {
+    height: 435px;
+    width: 369px;
+    margin-left: 10px;
+    background-image: url("./Images/Img-Placeholder11.png");
+  }
+
+  .art-container2 {
+    height: 435px;
+    width: 369px;
+    margin-right: 143px;
+    background-image: url("./Images/Mask-Group.png");
+  }
+
+  .art-container > .mycard:hover,
+  .art-container1 > .mycard:hover,
+  .art-container2 > .mycard:hover {
+    opacity: 0;
+  }
```

Comment:
- [ ] Hey Arnold, I highly advise you to carefully make changes as adding one thing might break others. So when I hover over the "See project" button the rest of the project description seems to be appearing which isn't the desired effect. See below 👇
  ![firefox_PCrLj6I2Nh](https://user-images.githubusercontent.com/104968236/236441003-61f4d2d6-da79-4754-b027-1167cf64e31a.gif)

- [ ] So here's how you could fix it:
    - [ ] 1st you should give the project cards the same class name as `project-card`. eg: `<div class="art-container1 project-card">` this also applies to the rest of the project cards.
    - [ ] 2nd Instead of adding the `:hover` to `.mycard` class you should add it to `.project-card` class. example below 👇
      ```
         .project-card:hover .mycard {
            opacity: 0;
          }
      ```
- It will then look like this:
  ![firefox_5lf79HMysv](https://user-images.githubusercontent.com/104968236/236442606-d7781960-6743-437e-b610-5d512ff95378.gif)

Reply:
noted thank you